models/engine/file_storage.py

```diff
@@ -68,3 +68,14 @@ def delete(self, obj=None):
     def close(self):
         """Deserialize JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Returns obj based on cls and id else None"""
+        obj = None
+        if type(cls) == str and type(id) == str:
+            obj = self.__objects.get(cls + '.' + id)
+        return obj
```

Comment:
Return result directly instead of through intermediate variable. If done this way, then an `else` will be required as done in the `DBStorage` version of this.